tests/sentry/auth/test_access.py

```diff
@@ -538,6 +538,13 @@ def test_superuser(self):
         result = self.from_request(request)
         assert result.has_permission("test.permission")
 
+    @patch("sentry.auth.access.is_active_superuser", return_value=True)
```

Comment:
this mock is not being respected :(

the real `is_active_superuser` function is still being called regardless if i mock using `sentry.auth.access.is_active_superuser` or `sentry.auth.superuser.is_active_superuser`

see the breakpoints

Reply:
in other tests i see Richard actually logged in as superuser, so i'm wondering if something weird is going on here

Reply:
but this test has an `is_active_superuser` mock and appears to be fine https://github.com/getsentry/sentry/blob/f03f11710b307ce7d7dbff661b6e7dfb218ecb0b/tests/sentry/api/test_base.py#L607